### PR TITLE
Remove xzhang

### PR DESCRIPTION
**Pull request by zxkjack123**
Move myself (@zxkjack123) from current student to alumni.

### CODE REVIEW

File: source/people/main.rst [hunk 1]
```diff
@@ -123,6 +115,7 @@ None at this time.
 
 Alumni
 -------------
+* Xiaokang Zhang, Visiting Student Researcher (2017-2019), University of Science and Technology of China
```
Comment (gonuke) [inline source/people/main.rst]: This list is alphabetical by family name.  Please move to the correct position

Comment (zxkjack123) [reply source/people/main.rst]: Done in https://github.com/CNERG/cnerg.github.com/pull/151/commits/e2ac79979b519d16d8a0b00c08568bef13c37a2b.

**Comment by gonuke** [comment]
Thanks @zxkjack123

**Comment by gonuke** [comment]
P.S. We'll miss you

**Pull request merged**